src/transformers/models/flava/configuration_flava.py

```diff
@@ -0,0 +1,611 @@
+# coding=utf-8
+# Copyright 2021 Meta Platforms authors and The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" FLAVA model configuration"""
+
+import copy
+import os
+from typing import Union
+
+from ...configuration_utils import PretrainedConfig
+from ...utils import logging
+
+
+logger = logging.get_logger(__name__)
+
+FLAVA_PRETRAINED_CONFIG_ARCHIVE_MAP = {
+    "flava-full": "https://huggingface.co/aps/flava-full/resolve/main/config.json",
+}
+
+
+class FLAVAImageConfig(PretrainedConfig):
+    r"""
+    This is the configuration class to store the configuration of a [`FLAVAImageModel`]. It is used to instantiate an
+    FLAVA model according to the specified arguments, defining the model architecture. Instantiating a configuration
+    with the defaults will yield a similar configuration to that of the FLAVA
+    [full](https://huggingface.co/aps/flava-full) architecture.
```

Comment:
```suggestion
    [aps/flava-full](https://huggingface.co/aps/flava-full) architecture.
```

Reply:
I would leave this as it is to avoid ambiguity in the meaning of the sentence itself.

Reply:
No, it is ambiguous as it doesn't show the real name of the checkpoint.